### PR TITLE
added whoami command

### PR DESCRIPTION
**Pull request by rishabh-vasudevan**
I have made another function in utils because the last function that printed the version took &Result<config> as its argument, so I made another function that does the same thing but with config:&Config as argument.

If there is anything that I should have done differently or something you would like me to add please let me know.

### CODE REVIEW

File: src/arguments.rs [hunk 1]
```diff
@@ -172,6 +173,10 @@ pub fn parse_arguments() -> ArgMatches<'static> {
             SubCommand::with_name(Other_commands::token.as_ref())
                 .about("Print a valid bearer token for the drogue cloud instance."),
         )
+        .subcommand(
+            SubCommand::with_name(Other_commands::whoami.as_ref())
```
Comment (jbtrystram) [inline src/arguments.rs:177]: who ami needs a `--token` argument that do the same as the `token` subcommand

File: src/main.rs [hunk 2]
```diff
@@ -48,6 +48,13 @@ fn main() -> Result<()> {
         openid::print_token(&config);
         exit(0);
     }
+    
+    if matches.is_present(Other_commands::whoami) {
+        println!("cluster adress : {}", config.drogue_cloud_url);
+        util::print_version_config(&config);
```
Comment (jbtrystram) [inline src/main.rs:54]:
```suggestion
        util::print_version(Ok(config));
```

**Review by jbtrystram — CHANGES_REQUESTED** [review]

**Comment by jbtrystram** [comment]
Also make sure to run `cargo fmt` on your code before submitting a PR. Don't pay attention to the "packit" failing tests

**Comment by jbtrystram** [comment]
@rishabh-vasudevan  are you planning to update this PR ?

**Comment by rishabh-vasudevan** [comment]
> @rishabh-vasudevan are you planning to update this PR ?

Yeah I am planning to work on this. I have my college exams till sunday, will work on it after they get over.

**Comment by jbtrystram** [comment]
Sure ! I don't want to pressure you in anyway, I just wanted to check in :) Good luck on your exams

**Pull request closed without merging**